news/supporting-the-future-of-rust-spotlight-on-teaching-initiatives.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Supporting the Future of Rust: Spotlight on Teaching Initiatives'
+byline: Rust Foundation Team
+description: >-
+  Learn how Rust-Edu, teach-rs, and other educational initiatives are shaping
```

Comment:
Let's change this to something like "Learn how various educational initiatives are shaping the future of learning Rust." As it stands here, we are just mentioning 2 of the 3 that are spotlit.